src/containerapp/azext_containerapp/tests/latest/test_containerapp_workload_profile_commands.py

```diff
@@ -30,10 +31,7 @@ def test_containerapp_env_workload_profiles_e2e(self, resource_group):
 
         location = "northcentralus"
 
-        self.cmd("az network vnet create -l {} --address-prefixes '14.0.0.0/16' -g {} -n {}".format(location, resource_group, vnet))
-        sub_id = self.cmd("az network vnet subnet create --address-prefixes '14.0.0.0/22' --delegations Microsoft.App/environments -n sub -g {} --vnet-name {}".format(resource_group, vnet)).get_output_in_json()["id"]
```

Comment:
why exclude subnet? ideally, we would want coverage for both vnet and non-vnet

Reply:
subnet tested in other test, removed to test without subnet